tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -196,6 +199,33 @@ const filterByDate = async (startDate, endDate) => {
   await (await $('#freetext')).click(); // blur the datepicker
 };
 
+const openSidebarFilter = async () => {
+  await (await sidebarFilterOpenBtn()).click();
+  return (await sidebarFilter()).waitForDisplayed();
+};
+
+const openSidebarFilterAccordion = async (selector) => {
+  await (await $(`${selector} .panel-heading`)).click();
+  return (await $(`${selector} .panel-collapse.show`)).waitForDisplayed();
+};
+
+const setSidebarFilterDate = async (fieldId, calendarIdx = 1, day = 'r1c2') => {
+  await (await $(fieldId)).waitForDisplayed();
+  await (await $(fieldId)).click();
+  await browser.pause(300); // Wait for animation to finish
```

Comment:
After I added the `commonElements.waitForPageLoaded()` in the `reports-sidebar-filter.wdio-spec.js` file I removed this `browser.pause(300)` and the test passed successfully.
Could you please re-check if you really need it?

Reply:
Oh wooooow, I ran it few times and haven't failed, Thanks!